rules/static.go

```diff
@@ -75,6 +152,23 @@ func (r *DefaultRuleRenderer) filterInputChain(ipVersion uint8) *Chain {
 		)
 	}
 
+	if r.KubeIPVSSupportEnabled {
+		// Check if packet belongs to forwarded traffic. (e.g. part of an ipvs connection).
+		// If it is, set endpoint mark and skip "to local host" rules below.
+		inputRules = append(inputRules,
+			Rule{
+				Action: ClearMarkAction{Mark: r.IptablesMarkEndpoint},
+			},
+			Rule{
+				Action: JumpAction{Target: ChainForwardCheck},
+			},
+			Rule{
+				Match:  Match().MarkNotClear(r.IptablesMarkEndpoint),
+				Action: ReturnAction{},
```

Comment:
Not sure this should be a return, since that will defer to the iptables policy, which might be DROP.  I think it should be the configured accept action `IptablesFilterAllowAction`.   WDYT?

Reply:
As discussed. Leave it for now.